wallet-sdk/src/main/kotlin/org/stellar/walletsdk/customer/Sep12.kt

```diff
@@ -116,6 +127,7 @@ internal constructor(
 
     val urlBuilder = URLBuilder(baseUrl)
     urlBuilder.appendPathSegments("customer")
+    urlBuilder.addParameter("transaction_id", transactionId)
```

Comment:
@philipliu @Ifropc same question here: shouldn't the transaction_id value go in the request body (aka populateMap) instead of a url parameter for this PUT request?